opm/input/eclipse/EclipseState/InitConfig/Equil.cpp

```diff
@@ -165,16 +178,44 @@ namespace Opm {
         return this->stress_zz_grad;
     }
 
-    bool StressEquilRecord::operator==(const StressEquilRecord& data) const {
-        return datum_depth == data.datum_depth &&
-               datum_posx == data.datum_posx &&
-               datum_posy == data.datum_posy &&
-               stress_xx == data.stress_xx &&
-               stress_xx_grad == data.stress_xx_grad &&
-               stress_yy == data.stress_yy &&
-               stress_yy_grad == data.stress_yy_grad &&
-               stress_zz == data.stress_zz &&
-               stress_zz_grad == data.stress_zz_grad;
+    double StressEquilRecord::stressXY() const {
+        return this->stress_xy;
+    }
+
+    double StressEquilRecord::stressXY_grad() const {
+        return this->stress_xy_grad;
+    }
+
+    double StressEquilRecord::stressXZ() const {
+        return this->stress_xz;
+    }
+    double StressEquilRecord::stressXZ_grad() const {
+        return this->stress_xz_grad;
+    }
+
+    double StressEquilRecord::stressYZ() const {
+        return this->stress_yz;
+    }
+    double StressEquilRecord::stressYZ_grad() const {
```

Comment:
brace

Reply:
> brace

Right.  I've pushed an update to address this.